### PR TITLE
Dependency version bump + BroadcastRegionJoin fix

### PR DESCRIPTION
**Pull request by fnothaft**
On top of #1360. Potential fix for the flagging unit test issues in #1324 and #1360. See https://github.com/bigdatagenomics/utils/pull/116. CC @heuermh @devin-petersohn

### CODE REVIEW

**Comment by fnothaft** [comment]
This will fail because the bigdatagenomics/utils#116 patch isn't in the 0.2.13-SNAPSHOT yet, but, it conceptually should work.

**Comment by heuermh** [comment]
Thanks for putting this together

**Comment by devin-petersohn** [comment]
Locally built #1324 on this and it builds correctly.

**Comment by fnothaft** [comment]
Updated to cut out the updates from #1360 that we don't want in 0.22.0. If this passes CI, I'll cut a 0.2.13 of utils tomorrow.

**Comment by fnothaft** [comment]
Whoops! Messed up the CI build for utils and we weren't pushing snapshots for the Spark 1 builds. Fixed that and am kicking off a utils trunk build now.

**Comment by fnothaft** [comment]
Jenkins, retest this please.

File: pom.xml [hunk 1]
```diff
@@ -28,8 +28,8 @@
     <hadoop-bam.version>7.7.0</hadoop-bam.version>
     <slf4j.version>1.7.21</slf4j.version>
     <bdg-formats.version>0.10.1</bdg-formats.version>
-    <bdg-utils.version>0.2.11</bdg-utils.version>
-    <htsjdk.version>2.5.0</htsjdk.version>
+    <bdg-utils.version>0.2.13-SNAPSHOT</bdg-utils.version>
```
Comment (devin-petersohn) [inline pom.xml]: Are we taking out the SNAPSHOT dependency once 0.2.13 is released?

**Comment by fnothaft** [comment]
Just cut org.bdgenomics.utils 0.2.13; waiting for the push to Maven Central.

**Comment by devin-petersohn** [comment]
Thanks @fnothaft!

**Comment by heuermh** [comment]
utils 0.2.13 is up, e.g. http://repo1.maven.org/maven2/org/bdgenomics/utils/utils-parent_2.10/0.2.13/

**Comment by fnothaft** [comment]
Thanks for the ping, @heuermh! Just pushed an update.

**Comment by heuermh** [comment]
Jenkins, retest this please.

**Comment by fnothaft** [comment]
Resolves #1455 now. Ping @heuermh for re-review.

**Comment by heuermh** [comment]
Thank you, @fnothaft!

**Pull request merged**